### ISSUE TITLE
Feature request - local filters vs global filters (conditionally show actions, observables, computeds)

### ISSUE THREAD

**Issue by akkayy** [kubk/mobx-log#23]
Currently, filters can be applied to show computeds, actions, observables.. but this is set within `configureMakeLoggable` which applies these filters globally. 

Is it possible to override these global filters, locally for specific stores. This is very useful when having multiple stores but when we want to apply different filters to each of them. 

Possible API could be when we mark a store loggable: 
```
makeLoggable(this, {
  filters: {
    computeds: true,
    actions: true,
    observables: false,
  }
})
```

Thanks bunch!

**Comment by kubk** [comment]
Thank you for the suggestion, I'll add it 👍

**Comment by kubk** [comment]
Released as 0.8.5

Here is an example how to do it: https://github.com/kubk/mobx-log#log-observables--computeds--actions-of-a-specific-store

I've chosen this type of configuration:

```js
    makeLoggable(this, {
      filters: {
        events: {
          computeds: true,
          observables: false,
          actions: false,
        },
      },
    });
```

It allows to separate computeds / observables / actions from store properties in the future:

```js
class Store {
  counter = 1;

  constructor() {
    makeAutoObservable(this);
    makeLoggable(this, {
      filters: {
        events: {
          computeds: true,
          observables: true,
          actions: false,
        },
        properties: {
          counter: true,
        }
      }
    })  
  }
}
```
